src/components/WindowEmergency.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+defineProps({
+  title: String,
+  description: String,
+  warning: String,
+  occurred: String,
+})
```

Comment:
```suggestion
defineProps({
  title: String,
  description: String,
  warning: String,
  occurred: String,
})
```
`String` -> `string`

---
TypeScriptなら
```ts
defineProps<{
  foo: bar,
}>();
```
とすると型がつくのでおすすめ

---
```ts
defineProps<{
  disastarInfo: DisastarInfo,
}>();
```
にして親コンポーネント側を`<WindowEmergency :disastarInfo="disasterInfo" />`にしたほうが素直かも